examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,640 @@
+# Analyzing Nginx Access Logs using Timeplus Proton 
+In a [recent blog post](https://www.timeplus.com/post/log-stream-analysis), we walked through three different ways of using Timeplus Proton to ingest and analyse a log file in real-time.
+
+I’ll explore a similar theme in the first part of this post: ad-hoc analysis of web traffic, in real-time, using Timeplus Proton. I will use a Node.js blog that is behind an Nginx web server for the analysis. 
```

Comment:
behind a Nginx, I think

Reply:
From the [source](https://www.nginx.com/about/the-nginx-name/):
> How to use the NGINX name (pronounced “engine x”):

So I'm using "an Nginx" because when it is vocalized as "an engine x", it's clear it starts with a vowel sound rather than a constant.

(Same goes for nVidia—it is pronounced ["en·vi·dee·uh"](https://www.google.com/search?q=nvidia+pronounce+in+english#ip=1)).

Reply:
Nice. I learned something new